packages/falter-berlin-tunnelmanager/files/up.sh

```diff
@@ -3,7 +3,11 @@
 interface="$1"
 
 # can be controlled using tunnelmanagers "-A" parameter
-prefix="$2"
+extra_args="$2"
+
+prefix=$(echo $extra_args | cut -d ' ' -f1)
+metric=$(echo $extra_args | cut -d ' ' -f2)
+lqm=$(echo $extra_args | cut -d ' ' -f3)
```

Comment:
I think your patch will break here. You should have quotes around `$extra_args`

Reply:
did.